jar-connector/src/main/java/com/sixsq/slipstream/connector/UsageRecorder.java

```diff
@@ -75,9 +75,8 @@ public static void insertEnd(String instanceId, String user, String cloud, List<
 			}
 
 			if(!hasRecorded(cloud, instanceId)){
-				logger.fine("Not recorded => avoiding inserting usage record END for "
+				logger.fine("Not recorded but still inserting usage record END for "
 						+ metrics + ", " + describe(instanceId, user, cloud));
-				return;
```

Comment:
An inconsistent transition will be detected without consequences. (only a INFO log).
See `initial` -> `stop` in ssclj/jar/src/main/clojure/com/sixsq/slipstream/ssclj/usage/state_machine.clj